light-client/src/light_client.rs

```diff
@@ -182,9 +182,8 @@ impl LightClient {
             // Trace the current height as a dependency of the block at the target height
             state.trace_block(target_height, current_height);
 
-            // If the trusted state is now at the height greater or equal to the target height,
-            // we now trust this target height, and are thus done :) [LCV-DIST-LIFE.1]
-            if target_height <= trusted_state.height() {
+            // If the trusted state is now at a height equal to the target height, we are done. [LCV-DIST-LIFE.1]
+            if target_height == trusted_state.height() {
```

Comment:
The case  `target_height` < `trusted_height` still needs to be spec'ed out I assume?

Reply:
It's already spec'ed out in a draft PR (#340) but implementing it is not a priority since the IBC relayer will in the end not need it.

Reply:
We might need some tests to make sure we error out properly when that case occurs, though.

Reply:
Do we still need to add somewhere logic for `target_height < trusted_height` case? We can just error for the moment in that case.

Reply:
Yes, I am on it :)

Reply:
Done